.github/workflows/virusScanAllAddons.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       VT_API_KEY: ${{ secrets.VT_API_KEY }}
       VT_API_LIMIT: ${{ vars.VT_API_LIMIT }}
       BRANCH_NAME: addVTURLs${{ github.run_number }}
-      BATCH_SIZE: 100
+      BATCH_SIZE: 10
```

Comment:
temporarily adjusted for testing, we can probably up this to 300 once testing is complete